python-package/xgboost/dask.py

```diff
@@ -71,6 +70,19 @@
 except ImportError:
     TrainReturnT = Dict[str, Any]  # type:ignore
 
+__all__ = [
+    "RabitContext",
+    "DaskDMatrix",
+    "DaskPartitionIter",
+    "DaskDeviceQuantileDMatrix",
+    "DaskScikitLearnBase",
+    "DaskXGBRegressor",
+    "DaskXGBClassifier",
+    "DaskXGBRanker",
+    "DaskXGBRFRegressor",
+    "DaskXGBRFClassifier",
+]
```

Comment:
Do we need the `train`, `predict` and `inplace_predict`?

Reply:
yeah, I'll add those